Sql/1v1_Battlemaster.sql

```diff
@@ -1,2 +1,4 @@
+DELETE FROM `creature_template` WHERE `entry` = 999990;
+DELETE FROM `creature_template` WHERE `entry` = 999991;

```

Comment:
those two lines can be combined:

http://www.azerothcore.org/wiki/Dealing-with-SQL-files#write-compact-code

everything else looks good

Reply:
done :)